### ISSUE TITLE
austin-tui seems to be unable to profile complex application

### ISSUE THREAD

**Issue by xmatthias** [P403n1x87/austin#58]
### Description

It seems impossible to profile larger applications with tui that require startup with subcommands.

```
austin-tui  ./project/main.py startexpensiveprocess --userdir ../ ...
```

This simply fails with the following error.

```
austin-tui: error: unrecognized arguments: --userdir ../ 
```

This suggests that somehow the arguments are not recognized to belong to the subcommand (most likely because it's 2 positional commands instead of the expected one)

**Expected behavior:** Profiling of the running python application

**Actual behavior:** not working

**Reproduces how often:** always

### Versions

You can get this information from copy and pasting the output of `austin
--version` from the command line. Also, please include the OS and what version
of the OS you're running.

austin 2.0.0
Archlinux

**Comment by P403n1x87** [comment]
Fixed by [austin-tui@v0.1.2](https://github.com/P403n1x87/austin-tui/releases/tag/v0.1.2).

**Comment by xmatthias** [comment]
Perfect, thanks a lot for the quick fix! :+1:
